BackofficeBundle/Resources/public/coffee/groupTree/NodeTreeView.coffee

```diff
@@ -0,0 +1,67 @@
+NodeTreeView = OrchestraView.extend(
+  tagName: 'ul'
+  events:
+    'click em.fa': 'toggleItemDisplay'
+    'click i': 'clickInput'
+  initialize: (options) ->
+    @initializer options
+    @loadTemplates [
+      'OpenOrchestraBackofficeBundle:BackOffice:Underscore/backToList',
+      'OpenOrchestraBackofficeBundle:BackOffice:Underscore/nodeTree',
+    ]
+    return
+
+  initializer: (options) ->
+    @options = options
+    @options.listUrl = appRouter.generateUrl('listEntities', entityType: options.entityType) if options.listUrl == undefined
+    @options.formView = 'editEntityTab'
+    @options.returnButton = true if !@options.domContainer
+    @options.domContainer = @$el if !@options.domContainer
+    if @options.roles == undefined
+      roles = {};
+      $.ajax
+        url: @options.html.links._role_list_node
+        method: "GET"
+        async: false
```

Comment:
@itkg-ngilain the first solution is impossible because the nodeTreeView is created by the panel loader, so I can only pass 1 element.

All right for the second solution